packages/page-anchor/src/Batch.tsx

```diff
@@ -54,22 +54,23 @@ function Batch(): React.ReactElement {
       return;
     }
 
-    const { compute_root, create_proof } = await import('mrklt');
+    const { construct } = await import('mrklt');
 
     // Processing files in sequence seems to be faster than in processing them in parallel
     // and it helps prevent OOM.
     for (const [f, i] of enumerate(files)) {
-      proofs[i].blake2s256 = await blake2sFile(f);
+      proofs[i].blake2b256 = await blake2b256File(f);
       setProofs(clone(proofs));
     }
 
-    const pl = pack32(proofs.map(p => p.blake2s256 as Uint8Array));
-    const root = compute_root(pl);
-    for (const [p, i] of enumerate(proofs)) {
-      p.proof = create_proof(i, pl);
+    const pl = pack32(proofs.map(p => p.blake2b256 as Uint8Array));
+    const [root, merkleproofs] = construct(pl);
+    assert(merkleproofs.length === proofs.length);
+    for (let i = 0; i < proofs.length; i++) {
+      proofs[i].proof = merkleproofs[i];
     }
     setProofs(proofs);
-    setRoot(root);
+    setRoot(new Uint8Array(root));
```

Comment:
For a technical reason the new merkle tree constuctor returns an array rather than a Uint8Array. I convert it to a Uint8Array here.